benchmarks/retrieval/retrieve.py

```diff
@@ -2,22 +2,26 @@
 
 Make sure to `pip install ir_measures` before running this script.
 """
-
+import csv
 import json
 import logging
 import os
 import time
+from pprint import pprint
 
 import configargparse
+from dotenv import load_dotenv
```

Comment:
Does `dotenv` need a dependency in requirements.txt?

Reply:
good catch